website/src/libs/api-v3/i18n.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'fs';
+import path from 'path';
+import _ from 'lodash';
+import shared from '../../../../common';
+
+export const localePath = path.join(__dirname, '/../../../../common/locales/');
+
+// Store translations
+export let translations = {};
+// Store MomentJS localization files
+export let momentLangs = {};
+
+// Handle differencies in language codes between MomentJS and /locales
+let momentLangsMapping = {
+  en: 'en-gb',
+  en_GB: 'en-gb', // eslint-disable-line camelcase
+  no: 'nn',
+  zh: 'zh-cn',
+  es_419: 'es', // eslint-disable-line camelcase
+};
+
+function _loadTranslations (locale) {
+  let files = fs.readdirSync(path.join(localePath, locale));
+
+  translations[locale] = {};
+
+  files.forEach((file) => {
+    if (path.extname(file) !== '.json') return;
+
+    // We use require to load and parse a JSON file
+    _.merge(translations[locale], require(path.join(localePath, locale, file))); // eslint-disable-line global-require
+  });
+}
+
+// First fetch English strings so we can merge them with missing strings in other languages
+_loadTranslations('en');
+
+// Then load all other languages
+fs.readdirSync(localePath).forEach((file) => {
+  if (file === 'en' || fs.statSync(path.join(localePath, file)).isDirectory() === false) return;
+  _loadTranslations(file);
+
+  // Merge missing strings from english
+  _.defaults(translations[file], translations.en);
+});
+
+// Add translations to shared
+shared.i18n.translations = translations;
+
+export let langCodes = Object.keys(translations);
+
+export let avalaibleLanguages = langCodes.map((langCode) => {
+  return {
+    code: langCode,
+    name: translations[langCode].languageName,
+  };
+});
+
+langCodes.forEach((code) => {
+  let lang = _.find(avalaibleLanguages, {code});
+
+  lang.momentLangCode = momentLangsMapping[code] || code;
+
+  try {
+    // MomentJS lang files are JS files that has to be executed in the browser so we load them as plain text files
+    // We wrap everything in a try catch because the file might not exist
+    let f = fs.readFileSync(path.join(__dirname, `/../../../node_modules/moment/locale/${lang.momentLangCode}.js`), 'utf8');
+
+    momentLangs[code] = f;
+  } catch (e) { // eslint-disable-lint no-empty
+    // TODO implement some type of error loggin?
+    // The catch block is mandatory so can't be removed
+  }
+});
+
+// Remove en_GB from langCodes checked by browser to avoid it being
+// used in place of plain original 'en' (it's an optional language that can be enabled only in setting)
+export let defaultLangCodes = _.without(langCodes, 'en_GB');
+
+// A map of languages that have different versions and the relative versions
+export let multipleVersionsLanguages = {
+  es: ['es-419', 'es-mx', 'es-gt', 'es-cr', 'es-pa', 'es-do', 'es-ve', 'es-co', 'es-pe',
+        'es-ar', 'es-ec', 'es-cl', 'es-uy', 'es-py', 'es-bo', 'es-sv', 'es-hn',
+        'es-ni', 'es-pr'],
+  zh: ['zh-tw'],
+};
+
+// Export en strings only, temporary solution for mobile
```

Comment:
Ok, I thought this code was used in some other way. Then maybe this should be removed, I think we can review it later. If you know where it's used because I couldn't find it